Sources/WebDriver/Requests.swift

```diff
@@ -462,6 +462,37 @@ public enum Requests {
         }
     }
 
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidexecute
+    public struct SessionScript: Request {
+        public var session: String
+        public var script: String
+        public var args: [String]
+
+            public var pathComponents: [String] { ["session", session, "execute"] }
+            public var method: HTTPMethod { .post }
+            public var body: Body { .init(script: script, args: args) }
+
+        public struct Body: Codable {
+            public var script: String
+            public var args: [String]
+        }
+    }
+
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidexecute_async
```

Comment:
We don't strictly need to have one struct per request url so consider just adding a `public var async: Bool` to the original struct and reusing it.